### PR TITLE
Refactor and minor cleanup in circuits

### PR DESCRIPTION
**Pull request by saleel**
## Motivation

Refactor and minor cleanup in circuits

## Change Summary

- Rename files to kebab case
- Rename circuit signals to make easier for devs
- Use camelCase in circuits
- Add comments to circuit variables

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: packages/circuits/src/aadhaar-verifier.circom [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+include "./helpers/sha.circom";
+include "./helpers/timestamp.circom";
+include "./extractor.circom";
+
+// Circuit to verify Aadhaar signature
+// n: RSA pubic key size per chunk
+// k: Number of chunks the RSA public key is split into
+// maxDataLength: Maximum length of the data
+template AadhaarVerifier(n, k, maxDataLength) {
+    signal input aadhaarData[maxDataLength];    // Aadhhar data padded (the data that is SHA hashed and signed)
```
Comment (Meyanis95) [inline packages/circuits/src/aadhaar-verifier.circom:13]: Just a small typo here
Aadhhar

File: packages/circuits/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "echo 'no-build'",
     "test": "npx ts-mocha test/**/*.test.ts --exit",
-    "format": "prettier --ignore-path .prettierignore --write \"**/*.+(js|ts|json)\"",
+    "format": "prettier --ignore-path ../../.prettierignore --write \"**/*.+(js|ts|json)\"",
```
Comment (Meyanis95) [inline packages/circuits/package.json:9]: `build` command should be added too, but I can do it after merge anyway

Comment (saleel) [reply packages/circuits/package.json:9]: Yes, was wondering why it was not there. I added to the shell script, let me add it.

File: packages/circuits/src/aadhaar-verifier.circom (hunk 1 above)
Comment (saleel) [reply packages/circuits/src/aadhaar-verifier.circom:13]: Fixed

File: packages/circuits/test/aadhaar-verifier.test.ts [hunk 3]
```diff
@@ -184,7 +187,7 @@ describe('Test QR Verify circuit', function () {
     assert(witness[3] === BigInt(expectedTimestamp))
   })
 
-  it('should output hash of pubkey', async () => {
+  it.only('should output hash of pubkey', async () => {
```
Comment (vuvoth) [inline packages/circuits/test/aadhaar-verifier.test.ts:190]: Remove .only

Comment (saleel) [reply packages/circuits/test/aadhaar-verifier.test.ts]: Fixed

**Review by Meyanis95 — APPROVED** [review]
LGTM

**Pull request merged**